contract_variable_quantity/views/abstract_contract_line.xml

```diff
@@ -8,10 +8,8 @@
             ref="contract.contract_abstract_contract_line_form_view"
         />
         <field name="arch" type="xml">
-            <xpath expr="//label[@for='quantity']" position="before">
-                <field name="qty_type" />
-            </xpath>
             <xpath expr="//field[@name='quantity']" position="after">
+                <field name="qty_type" />
```

Comment:
Please don't change this position, as it doesn't make sense to put the quantity before the quantity type

Reply:
Absolutely right @pedrobaeza , I changed the position field to show it before quantity.

![1](https://github.com/OCA/contract/assets/82393040/3cae937e-7d45-4d42-abb4-cccd0fceadc4)